### ISSUE TITLE
Ctrl + click doesn't work anymore (Typescript)

### ISSUE THREAD

**Issue by rexdavid** [microsoft/vscode#154787]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.69.0
- OS Version: Windows 11 , remote dev on Linux- PopOS

Steps to Reproduce:

> Ctrl + click doesn't work anymore with latest update
> Error squiggly lines do not show up anymore.

Note: reverting back to 1.68 fixed this issue

Language : Typescript
Windows Host , remote development on linux- PopOS

logs:
Unable to retrieve document from URI 'file:///home/user/Mounts/devNFS/app/src/controllers/modules/checkOrders/modifyOrder.ts': Error: Unable to retrieve document from URI 'file:///home/user/Mounts/devNFS/app/src/controllers/modules/checkOrders/modifyOrder.ts'
    at I.getDocument (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:87:11047)
    at d.provideCodeLenses (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:87:38149)
    at /home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:87:68077
    at te._withAdapter (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:87:66659)
    at te.$provideCodeLenses (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:87:68055)
    at s._doInvokeHandler (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:95:13708)
    at s._invokeHandler (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:95:13390)
    at s._receiveRequest (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:95:12107)
    at s._receiveOneMessage (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:95:10853)
    at /home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:95:8957
    at b.invoke (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:145)
    at v.deliver (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:2266)
    at p.fire (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:1844)
    at a.fire (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:69:18976)
    at /home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:111:34679
    at b.invoke (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:145)
    at v.deliver (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:2266)
    at p.fire (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:1844)
    at a.fire (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:69:18976)
    at r._receiveMessage (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:69:23563)
    at /home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:69:21097
    at b.invoke (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:145)
    at v.deliver (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:2266)
    at p.fire (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:1844)
    at p.acceptChunk (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:69:15807)
    at /home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:69:14937
    at b.invoke (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:145)
    at v.deliver (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:2266)
    at p.fire (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:61:1844)
    at f._processReadQueue (/home/user/.vscode-server/bin/92d25e35d9bf1a6b16f7d0758f25d48ace11e5b9/out/vs/workbench/api/node/extensionHostProcess.js:111:21245)

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.69.0. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by mjbvz** [comment]
Does this reproduce in the [latest VS Code insiders build](https://code.visualstudio.com/insiders/) with all [extensions disabled](https://github.com/microsoft/vscode/wiki/Performance-Issues#run-with-extensions-disabled)? (except remote)

**Comment by rexdavid** [comment]
Yes, this exists, however I see the error squiggly lines in the insiders build but ctrl+ click does not take to the right line, when error line numbers are displayed in the terminal using ( tsc build watch).

**Comment by vaimo-wilko** [comment]
I can currently repeat the same error with VS Code 1.69.2 (Commit: 3b889b090b5ad5793f524b5d1d39fda662b96a2a) on macOS 12.4 (Monterey) using the "Remote - Containers" extension to work inside a local docker container on a `.ts` file.

I noticed this also when trying to "Rename Symbol" (pressing F2 on a variable to rename it), but realised later that after startup (or reload of vscode) renaming with F2 works fine until I cmd-click on something that causes the error. After that rename no longer works, giving me the same error everytime.

The error message and call stack can be seen in the "Log (Remote Extension Host)" Output, and once vscode gets into this state, even hovering over a word in the code triggers the exception.'

(Btw, I disabled all remote extensions and all local extension (except the Remote extensions) and the issue remains the same)
